includes/class-comment-type.php

```diff
@@ -25,14 +25,20 @@ final class Comment_Type {
 	 */
 	public $label;
 
-
 	/**
 	 * Name of the comment type. Usually plural.
 	 *
 	 * @var string $singular
 	 */
 	public $singular;
 
+	/**
+	 * Single Character Emoji Representation of the Comment Type. Optional.
+	 *
+	 * @var string $icon
+	 */
+	public $icon;
```

Comment:
what about adding dashicons instead? https://developer.wordpress.org/resource/dashicons/

Reply:
Somewhat limited in their expandability though.

Reply:
That’s true.

Reply:
Maybe better off just adding a filter and people can change it